app.js

```diff
@@ -11,14 +11,16 @@ app.all('/api', (req, res) => {
 
 app.get('/api/topics', getTopics)
 app.get('/api/articles/:article_id', getArticleById)
-app.get('/api/articles/notAnId')
+// app.get('/api/articles/notAnId')
+app.patch('/api/articles/:article_id', addArticleVotes)
 
 
 app.all('/*', (req, res) => {
-    res.status(404).send( { msg: "PATH REQUESTED NOT FOUND" } )
+    res.status(404).send(( { msg: "PATH REQUESTED NOT FOUND" } ))
```

Comment:
Not hurting anyone - but you shouldn't need the additional layer of parentheses here